dspace/config/modules/elsevier-sciencedirect.cfg

```diff
@@ -0,0 +1,17 @@
+api.key = ${elsevier.api.key}
+
+api.article.url=http://api.elsevier.com/content/article
+ui.article.url=http://www.sciencedirect.com/science/article
+api.entitlement.url=http://api.elsevier.com/content/article/entitlement
+api.scidir.url=http://api.elsevier.com/content/search/scidir
+entitlement.check.enabled=false
+metadata.field.pii = elsevier.identifier.pii
+metadata.field.doi = dc.identifier
+
+embed.display=false
+embed.display.width=700
+embed.display.height=500
```

Comment:
Thanks for this advice

Best regards,

Letitia Mukherjee
Market Development Manager Sharing Platforms
Mob: +31 6 13 98 98 71

From: Tim Donohue [mailto:notifications@github.com]
Sent: 26 January 2016 16:02
To: DSpace/DSpace
Cc: Mukherjee, Letitia (ELS-AMS)
Subject: Re: [DSpace] DS-2877 Import of ScienceDirect metadata including embargo and linking to or embedding of the final version (#1162)


In dspace/config/modules/elsevier-sciencedirect.cfg<https://github.com/DSpace/DSpace/pull/1162#discussion_r50845233>:

> @@ -0,0 +1,17 @@

> +api.key = ${elsevier.api.key}

> +

> +api.article.url=http://api.elsevier.com/content/article

> +ui.article.url=http://www.sciencedirect.com/science/article

> +api.entitlement.url=http://api.elsevier.com/content/article/entitlement

> +api.scidir.url=http://api.elsevier.com/content/search/scidir

> +entitlement.check.enabled=false

> +metadata.field.pii = elsevier.identifier.pii

> +metadata.field.doi = dc.identifier

> +

> +embed.display=false

> +embed.display.width=700

> +embed.display.height=500

Please add inline comments to this configuration file to describe all of the above configurations, e.g. what each setting does, possible values (if any), etc. While they obviously also should be documented elsewhere, this is just our best practice for DSpace configurations.

—
Reply to this email directly or view it on GitHub<https://github.com/DSpace/DSpace/pull/1162/files#r50845233>.

________________________________

Elsevier B.V. Registered Office: Radarweg 29, 1043 NX Amsterdam, The Netherlands, Registration No. 33156677, Registered in The Netherlands.